pylinter.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+directories="linux MacOS windows"
+pass=0
+fail=0
+
+function prepare_venv() {
+    VIRTUALENV=$(which venv)
+    if [ $? -eq 1 ]; then
+        
+        VIRTUALENV=$(which venv-3)
+    fi
+
+    ${VIRTUALENV} python3 -m venv mybenjienv && source myenv/bin/activate && python3 "$(which pip3)" install pycodestyle
```

Comment:
Since you are naming your venv `mybenjienv`, `source`  should also use the same name and not `myenv`